composer.json

```diff
@@ -61,7 +71,7 @@
         "emanueleminotto/twig-cache-bundle": "~1.0",
         "incenteev/composer-parameter-handler": "~2.0",
         "friendsofsymfony/http-cache-bundle": "~1.0",
-        "friendsofsymfony/rest-bundle": "2.0.x-dev",
+        "friendsofsymfony/rest-bundle": "2.0.x-dev as 1.5",
```

Comment:
yep, it need to stay like that as long as Sylius will support 2.0.x-dev this same with "sylius/resource-bundle"